R/plotDMN.R

```diff
@@ -23,11 +23,15 @@
 #' @name plotDMN
 #'
 #' @examples
+#' library("bluster")
 #' data(dmn_se, package = "mia")
-#' names(metadata(dmn_se))
+#' tse_dmm <- cluster(dmn_se, name = "DMM",
+#'                    DmmParam(k = 1:4, type = "laplace"),
+#'                    MARGIN = "samples", full = TRUE)
```

Comment:
How long this takes? `dmn_se` object already has DMM calculated --> you could just plot it without actually calculating it (This removes bluster also from suggests)

Reply:
Good point. On the other hand I think we could show how one can cluster some new data with DMM because it is such a common and customized method for microbiome sample clustering? The example data set could be something else than dmn_se and small data set might be preferred to have fast calculation.

Reply:
The issue is that `dmn_se` has data in `metadata(tse)$dmm` instead of `metadata(tse)$DMM$dmm`. This creates a deprecated warning and the check fails.
I will change the data to put dmm in the correct place

Reply:
Sorry @antagomir I didn't see you reply, do you want me to change the example even if it adds bluster dependency? Or do I leave it as is?

Reply:
Do you mean it adds bluster dependency in OMA?

Reply:
Using bluster here adds a dependency in miaViz.
We already have a better, more complete example in OMA that uses bluster and miaViz.
So the example in miaViz should stay fairly simple.